modules/7.2.5/module.yaml

```diff
@@ -0,0 +1,23 @@
+schema_version: 1
+
```

Comment:
@dbkreling FYI, I missed this earlier, but for newer versions of cekt this needs:

version: "1.0"

All modules require the version tag. You won't have a problem with cekit-3.2.x, but anything after that will give you an error.